packages/lesswrong/lib/collections/posts/custom_fields.js

```diff
@@ -72,8 +72,9 @@ addFieldsDict(Posts, {
   title: {
     order: 10,
     placeholder: "Title",
+    label: "Title",
     control: 'EditTitle',
-    editableBy: [Users.owns, 'sunshineRegiment', 'admins']
+    editableBy: ['sunshineRegiment', 'admins']
```

Comment:
Eek, no, this means users can no longer edit the titles of their posts.

Reply:
Sorry, that was debris while trying to generally experiment with "can I make the editableBy function do anything on purpose". Reset to it's original value now